### ISSUE TITLE
broken when using nuxt/components together

### ISSUE THREAD

**Issue by odanado** [nuxt/vite#13]
Fix `test/fixture/nuxt.config.js`

```
$ git diff test
diff --git a/test/fixture/nuxt.config.js b/test/fixture/nuxt.config.js
index f2d6f7e..af76cca 100644
--- a/test/fixture/nuxt.config.js
+++ b/test/fixture/nuxt.config.js
@@ -1,5 +1,6 @@
 import viteModule from '../../src'
 export default {
+  components: true,
   buildModules: [
     viteModule
   ],
```

I got an error.
```
$ /Users/odan/source/github.com/nuxt/vite/node_modules/.bin/jest
  console.info
    Discovered Components: test/fixture/.nuxt/0o1wutbk/components/readme.md

      at node_modules/@nuxt/components/dist/index.js:230:13


 RUNS  test/index.test.ts
 > test/fixture/.nuxt/0o1wutbk/index.js:15:7: error: No matching export for import "default"
    15 │ import nuxt_plugin_plugin_d00e1886 from 'nuxt_plugin_plugin_d00e1886' // Source: ./components/plugin.js (mode: 'all')
       ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~~

 > test/fixture/.nuxt/0o1wutbk/index.js:15:7: error: No matching export for import "default"
    15 │ import nuxt_plugin_plugin_d00e1886 from 'nuxt_plugin_plugin_d00e1886' // Source: ./components/plugin.js (mode: 'all')
[error] Client compiled with errors: Build failed with 1 error:
test/fixture/.nuxt/0o1wutbk/index.js:15:7: error: No matching export for import "default"
  test/fixture/.nuxt/0o1wutbk/index.js:15:7: error: No matching export for import "default"
  at failureErrorWithLog (node_modules/esbuild/lib/main.js:1171:15)
  at buildResponseToResult (node_modules/esbuild/lib/main.js:907:32)
  at node_modules/esbuild/lib/main.js:1002:20
  at node_modules/esbuild/lib/main.js:553:9
  at handleIncomingPacket (node_modules/esbuild/lib/main.js:642:9)
  at Socket.readFromStdout (node_modules/esbuild/lib/main.js:520:7)
  at Socket.emit (node:events:376:20)
  at addChunk (node:internal/streams/readable:305:12)
  at readableAddChunk (node:internal/streams/readable:280:9)
  at Socket.Readable.push (node:internal/streams/readable:219:10)
  at Pipe.onStreamRead (node:internal/stream_base_commons:192:23)
[error] Server compiled with errors: Build failed with 1 error:
test/fixture/.nuxt/0o1wutbk/index.js:15:7: error: No matching export for import "default"
  test/fixture/.nuxt/0o1wutbk/index.js:15:7: error: No matching export for import "default"
  at failureErrorWithLog (node_modules/esbuild/lib/main.js:1171:15)
  at buildResponseToResult (node_modules/esbuild/lib/main.js:907:32)
  at node_modules/esbuild/lib/main.js:1002:20
  at node_modules/esbuild/lib/main.js:553:9
  at handleIncomingPacket (node_modules/esbuild/lib/main.js:642:9)
  at Socket.readFromStdout (node_modules/esbuild/lib/main.js:520:7)
  at Socket.emit (node:events:376:20)
  at addChunk (node:internal/streams/readable:305:12)
  at readableAddChunk (node:internal/streams/readable:280:9)
  at Socket.Readable.push (node:internal/streams/readable:219:10)
  at Pipe.onStreamRead (node:internal/stream_base_commons:192:23)
```

branch: 6a3bd49055d2a3a658c2e06c18e7debb20a1b2c1

**Comment by danielroe** [comment]
A much better issue - but see #12 - and fix at nuxt/components#157.

**Comment by odanado** [comment]
Thank you.
It appears that #12 was created while this issue was being created.

**Comment by pi0** [comment]
Should be fixed in 0.0.5
